apollo-gradle-plugin/gradle.properties

```diff
@@ -1,4 +1,3 @@
 POM_ARTIFACT_ID=apollo-gradle-plugin
 POM_NAME=Apollo Gradle Plugin
 POM_DESCRIPTION=Gradle plugin for generating java/kotlin classes for graphql files
-POM_PACKAGING=jar
```

Comment:
This is required to not override the default `<packaging>pom</packaging>` for the plugin marker